### DISCUSSION TITLE
DwC Observation to TERN NDES - modelling cookbook use case

### DISCUSSION THREAD

**Discussion by gwittich** [ternaustralia/ontology_tern#47] [Modelling]
Please include a mapping of:
  DwC Observation to TERN NDES
in the cookbook as a priority.

**Comment by gwittich** [comment]
Discussion point "DwC sample data #53" Contains two datasets. The dataset "2) The Onychophora around Perth dataset is a larger dataset that includes both material samples and human observations from a number of different sources. 
Onychophora around Perth records-2021-11-30.csv" is suitable for this part of the Cookbook. ie. it is a DwC Observation dataset

**Reply by smguru** [reply]
@gwittich @edmondchuc is working on the cookbook for a given example, probably will update in the first week of Jan,
